lib/src/file_descriptor.dart

```diff
@@ -134,7 +134,7 @@ class _StringFileDescriptor extends FileDescriptor {
       Stream.fromIterable([utf8.encode(_contents)]);
 
   @override
-  Future _validate(String prettyPath, List<int> actualContents) {
+  void _validate(String prettyPath, List<int> actualContents) {
     var actualContentsText = utf8.decode(actualContents);
     if (_contents == actualContentsText) return null;
```

Comment:
Can this omit the value?

```suggestion
    if (_contents == actualContentsText) return;
```

Reply:
Sure, updated all of these